serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/controller/ServiceRecordController.java

```diff
@@ -150,8 +163,14 @@ private void addServiceRecords(EntityInfo entityInfo, Entity entity, Authenticat
         Optional<ServiceRecord> eduServiceRecord = serviceRecordResponseHandler(serviceRecordFactory.createEduServiceRecord(orgnr), entity);
         eduServiceRecord.ifPresent(r -> entity.getServiceRecords().add(r));
 
-        Optional<ServiceRecord> fiksServiceRecord = serviceRecordResponseHandler(serviceRecordFactory.createFiksServiceRecord(orgnr), entity);
-        fiksServiceRecord.ifPresent(r -> entity.getServiceRecords().add(r));
+        Optional<FiksWrapper> fiksWrapper = serviceRecordFactory.createFiksServiceRecord(orgnr);
+        if (fiksWrapper.isPresent()) {
+            Optional<ServiceRecord> fiksServiceRecord = serviceRecordResponseHandler(Optional.of(fiksWrapper.get().getServiceRecord()), entity);
+            if (fiksServiceRecord.isPresent()) {
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS2583)